haiku-apps/ultraencode/ultraencode-1.0.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="CD audio extraction program"
+DESCRIPTION="UltraEncode is a CD audio and extraction program. It is \
+designed to allow the quick and easy extraction of CD audio tracks and \
+compression of audio tracks. UltraEncode currently allows compression \
+of files using MPEG Layer 3 encoding."
+HOMEPAGE="https://github.com/HaikuArchives/UltraEncode"
+COPYRIGHT="unknown"
+LICENSE="unknown"
```

Comment:
Licenses:

- [GNU LGPL v2.1](https://github.com/HaikuArchives/UltraEncode/blob/a11c3cfbe4d44208e8869fa614fe1d523cc2f9b1/mp3-add-ons/encoder/blade/codec.h#L11-L14)
- [GNU GPL v2](https://github.com/HaikuArchives/UltraEncode/blob/a11c3cfbe4d44208e8869fa614fe1d523cc2f9b1/Source/id3ren.c#L6-L9)

And yea, "Unknown" still have to stay since a large portion of the program is unlicensed.

Copyrights: https://github.com/HaikuArchives/UltraEncode/search?q=Copyright&unscoped_q=Copyright

Reply:
@alaviss Hehe didn't think about searching in the source... I'll remember to search in the source next time 😄 

But I can update the copyrights info. Thanks for helping!